ambry-api/src/main/java/com.github.ambry/store/MessageReadSet.java

```diff
@@ -36,7 +38,14 @@
   long writeTo(int index, WritableByteChannel channel, long relativeOffset, long maxSize) throws IOException;
 
   /**
-   * Returns the total number of messages in this set
+   * Write all messages in this set to the give {@link AsyncWritableChannel}
```

Comment:
Add asynchronously to the docs to indicate that this method returns immediately after the write is initiated?

Reply:
Added in comment:
```
   * This method is intend to write prefetched data from {@link MessageReadSet} to {@link AsyncWritableChannel}. Data
   * should be ready in memory(no blocking call) before write to {@link AsyncWritableChannel} asynchronously. Callback is
   * called when the entire batch of writes succeeds or fails.
```